lib/src/package_config.dart

```diff
@@ -158,7 +141,6 @@ class PackageConfig {
   Map<String, Object?> toJson() => {
     'configVersion': configVersion,
     'packages': packages.map((p) => p.toJson()).toList(),
-    'generated': generated?.toUtc().toIso8601String(),
```

Comment:
... which would make this:
```dart
      if (generated != null) 'generated': generated.toUtc().toIso8601String(),
```

But, on the other hand, if it's a feature in the Pub code that Pub never uses, it's basically dead code, so it'd be better to get rid of it.

(Do consider using `package:package_config` instead of your own classes. If we change things in the format in the future, we'll only need to fix one place. Or tell me why it's not usable enough.)

Reply:
I remember trying to use package_config but being frustrated with something. But I cannot remember what it was.

Maybe it was that I could not get to the raw rootUri property 

https://pub.dev/documentation/package_config/latest/package_config_types/Package/root.html is always resolved

Also I cannot get at the uri that a package config is resolving its packages against. (the baseUri)

Not sure anymore...